install/traf_tools_setup.sh

```diff
@@ -43,6 +43,7 @@
 #   common services such as naming, configuration management, synchronization, 
 #   and group services.
 # Thrift: Communications and data serialization tool
+# Maven: Build tool that is only installed if compatible version does not exist
```

Comment:
This was a mistake.  I was testing the code where Maven was already installed.  Removed the commented lines.